### PR TITLE
add support for named repository directories in config

### PR DESCRIPTION
**Pull request by vojtabiberle**
Hi,
I added support for named repository directories. Sometimes you have more directories for tests and VistualPHPUnit mixt all dirs/files in FileSelector.

Now is possible to extend config array and name this directories.

PS: I know my JS is not great, but I want let your way of configuration directories working togather with my new way.

### CODE REVIEW

File: app/controller/FileList.php [hunk 1]
```diff
@@ -18,7 +18,16 @@ public function index($request) {
         $test_directories = \app\lib\Library::retrieve('test_directories');
         $valid_dir = false;
         foreach ( $test_directories as $test_directory ) {
-            if ( strpos($dir, realpath($test_directory)) === 0 ) {
+            if(isset($test_directory['path']))
+            {
```
Comment (vojtabiberle) [inline app/controller/FileList.php]: Nice opinion, but I need two different behaviors for two different scenarios.

if $test_directory will be array
realpath($test_directory) fails

if $test_directory will be string
realpath($tesst_directory['path']) made notice

**Comment by localgod** [comment]
Hi
I'm taking over the maintenance on the project. As this is pretty old pull request I wanner know if this is still relevant or I can disregard it.

**Comment by vojtabiberle** [comment]
Hi,
right now I change my work so I personally don't use VisualPHPUnit right now (but I want use it in future) and in my old work people still use this change, so they will be pleased, if you merge my changes or if you made another functionality in same way as me.

**Comment by localgod** [comment]
This is added/fixed in the upcoming version 3.0. I will close the pull request as i conflicts with the old 2.* branch.

**Pull request closed without merging**